### PR TITLE
Made GTFO supports coreprotect.

### PR DESCRIPTION
**Pull request by ghost**
Made GTFO uses coreprotect if is loaded and enabled.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java [hunk 1]
```diff
@@ -42,18 +43,34 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
+            plugin.rb.rollback(player.getName());
+
         }
-        catch (NoClassDefFoundError ex)
+
+        if (getServer().getPluginManager().isPluginEnabled("CoreProtect"))
         {
+            try
+            {
+                server.dispatchCommand(sender, "co rollback t:1d r:#global u:" + player.getName());
```
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java:67]: We should not be using a dispatch command. Thats generally considered a big no-no

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java [hunk 2]
```diff
@@ -42,18 +43,34 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
+            {
+            }
+
+            // Rollback
+            plugin.rb.rollback(player.getName());
+
         }
-        catch (NoClassDefFoundError ex)
+
+        if (getServer().getPluginManager().isPluginEnabled("CoreProtect"))
```
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java:63]: Can we change this to an If-Else statement please?

**Review by Wild1145 — CHANGES_REQUESTED** [review]

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java (hunk 1 above)
Comment (JeromSar) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java:67]: Yes, this is in the contributing guidelines. Coreprotect should get a bridge, so TFM can interact with its API.

**Comment by ghost** [comment]
@Wild1145 This has been superseded by #2067

**Pull request closed without merging**